clients/linux/src/app.rs

```diff
@@ -603,7 +603,17 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
+                        Ok(f) => {
+                            lb.gui.win.set_title(&f.name);
```

Comment:
The name is already stored in the `name` variable. Let's use that here, and then move this statement out of (above) this match statement.